packages/app-content-pages/src/shared/components/ZooHeaderWrapper/ZooHeaderWrapperContainer.js

```diff
@@ -1,121 +1,63 @@
-    return this.redirect(url)
-  }
-
-  signOut() {
-    return auth.signOut()
-      .then(() => {
-        this.props.user.clear()
-        // resetting the alreay seen subjects during the session tracking should move
-        // once we refactor the UPP and User resource tracking in the classifier
-        // Current implementation in classifier is possibly buggy (see discussion https://github.com/zooniverse/front-end-monorepo/discussions/2362)
-        // likely a user id prop will be passed to the classifier and that will be reacted to with an effect hook
-        /// which would reset the subject already seen tracking
-        // I needed to guarantee that this happened on sign out only so that's why this is here for now
-        const seenThisSession = (window) ? window.sessionStorage.getItem("subjectsSeenThisSession") : null
-
-        if (seenThisSession) {
-          window.sessionStorage.removeItem("subjectsSeenThisSession")
-        }
-      })
-  }
-
-  unreadMessages() {
-    const { user } = this.props
+export function signOut(user) {
+  user.clear()
+  // resetting the alreay seen subjects during the session tracking should move
+  // once we refactor the UPP and User resource tracking in the classifier
+  // Current implementation in classifier is possibly buggy (see discussion https://github.com/zooniverse/front-end-monorepo/discussions/2362)
+  // likely a user id prop will be passed to the classifier and that will be reacted to with an effect hook
+  /// which would reset the subject already seen tracking
+  // I needed to guarantee that this happened on sign out only so that's why this is here for now
+  const seenThisSession = (window) ? window.sessionStorage.getItem("subjectsSeenThisSession") : null
 
-    return user?.personalization?.notifications?.unreadConversationsIds.length
-  }
-
-  unreadNotifications() {
-    const { user } = this.props
-    
-    return user?.personalization?.notifications?.unreadNotificationsCount
-  }
-
-  render() {
-    return (
-      <ZooHeader
-        {...this.props}
-        register={this.openRegisterModal}
-        signIn={this.openSignInModal}
-        signOut={this.signOut}
-        unreadMessages={this.unreadMessages()}
-        unreadNotifications={this.unreadNotifications()}
-        user={this.createUserProp()}
-      />
-    )
+  if (seenThisSession) {
+    window.sessionStorage.removeItem("subjectsSeenThisSession")
   }
+  auth.signOut()
 }
 
-ZooHeaderWrapperContainer.propTypes = {
-  router: shape({
-    asPath: string,
-    pathname: string,
-    push: func
-  }),
-  user: shape({
-    clear: func,
-    display_name: string,
-    login: string,
-    isLoggedIn: bool
-  })
+function ZooHeaderWrapperContainer(props) {
+  const { user } = useStore()
+  const router = useRouter()
+  const onSignOut = useCallback(() => signOut(user), [user])
+  const { display_name, login } = user
+  const userProp = user.isLoggedIn ? { display_name, login } : {}
```

Comment:
For the lib-react-components ZooHeader MainNavList `isAdmin={user?.admin && isAdmin}`, specifically `user?.admin` to be defined the following suggestion would be needed:

```suggestion
  const { admin, display_name, login } = user
  const userProp = user.isLoggedIn ? { admin, display_name, login } : {}
```

Or delete `user?.admin` from  lib-react-components ZooHeader MainNavList.